vscode/webviews/components/modelSelectField/ModelSelectField.module.css

```
@@ -25,26 +28,30 @@
 
     & > svg {
         opacity: 0.5;
-        margin-top: -1px
     }
 }
 
+button > .model-title-with-icon .model-provider {
+    /* HACK(sqs): Hide the provider name from the button (show it only in the dropdown). */
+    display: none;
+}
+
 .badge  {
     margin-left: auto;
     line-height: 1;
-    letter-spacing: -0.025rem;
+    letter-spacing: -0.02rem;
+    font-size: 90%;
```

```suggestion
    font-size: 11px;
    font-weight: 500;
```

or `calc(11/13 * 100%)` if you want it to be relative? (any reason we don't just hard code a size?)